### PR TITLE
hotfix/1/SC-7434 - qr code name of username fix

### PR DESCRIPTION
**Pull request by ABialoskorski**
# Description
fixed display of user name on qr code printsheets

## Links to Tickets or other pull requests
- https://ticketsystem.schul-cloud.org/browse/SC-7434

## Screenshots of UI changes
image

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by mwolowyk-cap — APPROVED** [review]

File: static/scripts/administration.js [hunk 1]
```diff
@@ -298,9 +298,7 @@ $(document).ready(() => {
 				printQRs(
 					users.map((user) => ({
 						href: user.registrationLink.shortLink,
-						title:
-							user.fullName
-							|| `${user.firstName} ${user.lastName}`,
+						title: `${user.firstName} ${user.lastName}`,
```
Comment (mwolowyk-cap) [inline static/scripts/administration.js:301]: why did you removed user.fullName, it will pick user.firstName user.lastName if user.fullName is empty

Comment (ABialoskorski) [reply static/scripts/administration.js:301]: yes true, but I think there is no more such thing like fullName

**Review by mwolowyk-cap — APPROVED** [review]

**Review by DanielRieckhof — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,9 @@ Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `
 
 ## Unreleased
 
-## [25.1.1] - 2020-10-15
+## [25.1.2] - 2020-10-20
```
Comment (DanielRieckhof) [inline CHANGELOG.md:12]: Please re-check, but version should be 25.1.5

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "schulcloud-client",
-	"version": "25.1.1",
+	"version": "25.1.2",
```
Comment (DanielRieckhof) [inline package.json:3]: Please re-check, but version should be 25.1.5

File: package-lock.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "schulcloud-client",
-	"version": "25.1.1",
+	"version": "25.1.2",
```
Comment (DanielRieckhof) [inline package-lock.json:3]: Please re-check, but version should be 25.1.5

**Review by DanielRieckhof — COMMENTED** [review]
Please check the version

**Review by DanielRieckhof — APPROVED** [review]

**Pull request merged**